pkg/virt-launcher/virtwrap/network/podnic_test.go

```diff
@@ -43,7 +42,9 @@ var _ = Describe("podNIC", func() {
 		return podnic, nil
 	}
 	BeforeEach(func() {
-		cacheFactory = fake.NewFakeInMemoryNetworkCacheFactory()
+		tmpDir, err := ioutil.TempDir("/tmp", "interface-cache")
```

Comment:
The same like above. In addition a `BeforeEach` directly below creates another tmpDir. So in this case we would even without a `:=` create two directories but only clean up one of them.

Reply:
Done